app/src/contract_event_recorders.py

```diff
@@ -161,6 +121,17 @@
 
     return bool(did_insert)
 
+def enqueue_order_update_for_transfer(event):
+    """
+    Enqueues a task to update orders' fill / available volume when a
+    Deposit/Withdraw event occurs.
+    """
+    user_addr = event["args"]["user"]
```

Comment:
You set these variables but didn't use them on line 133?

Reply:
Good catch!

Reply:
Fixed.